crates/depguard-render/README.md

```
@@ -0,0 +1,146 @@
+- **Location**: `Cargo.toml:15`
+- **Message**: Dependency `serde` uses wildcard version `*`
+
+### Warning: Path Without Version (`deps.path_requires_version`)
+
+- **Location**: `crates/my-crate/Cargo.toml:8`
+- **Message**: Path dependency `my-other-crate` should declare a version
+```
+
+### GitHub Actions
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)